src/modelbench/hazards.py

```diff
@@ -57,12 +58,23 @@ def __repr__(self):
 
 
 class SafeHazard(HazardDefinition, ABC):
+
+    def __init__(self):
+        # We expose these so that a BenchmarkDefinition can look into the specifics
+        # of what got into scoring calculations
+        super().__init__()
+        self.num_scored_items = 0
+        self.num_safe_items = 0
+
     def score(self, sut_scores: Mapping[str, TestRecord]) -> "HazardScore":
         test_scores: dict[str, ValueEstimate] = {}
         num_exceptions = 0
+        self.num_scored_items = 0
```

Comment:
If this gets called twice, which could well happen with different SUTs in the same run, I think the instance variables will be off. Note also that it isn't guaranteed that the number of scored items will be the same across all SUTs. E.g., if there's an exception with some item for one SUT but not another.

Reply:
Moved into HazardScore.